### PR TITLE
ci: temporarily remove race detection from travis

### PR DESCRIPTION
**Pull request by ricellis**
## PR summary

_Temporarily_ disable race checking because new generated test code is causing CI failures that are blocking updates.

**Note: An existing issue is [required](https://github.com/IBM/cloudant-go-sdk/blob/master/CONTRIBUTING.md#PRs) before opening a PR.**

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?

Travis CI fails because of races in new generated test code.

## What is the new behavior?

Disable race detection in Travis CI until we can get a fix from upstream for the generated test code.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

The problem has been reported upstream and this change will be reverted as soon as a fix is available.

### CODE REVIEW

**Review by mojito317 — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ install:
 - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.28.2
 
 script:
-- go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+- go test ./... -coverprofile=coverage.txt -covermode=atomic
```
Comment (vmatyusGitHub) [inline .travis.yml:30]: In this case the `set` is enough as covermode. 
```suggestion
- go test ./... -coverprofile=coverage.txt -covermode=set
```
This coverage calculation is enough for our project purpose, temporarily.

**Comment by bessbd** [comment]
Should we open a ticket for reverting this change?

**Review by vmatyusGitHub — APPROVED** [review]

**Comment by ricellis** [comment]
@bessbd I just opened https://github.com/IBM/cloudant-go-sdk/pull/59 straight away

**Comment by bessbd** [comment]
> @bessbd I just opened #59 straight away

Awesome, thank you @ricellis !

**Pull request merged**